packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -179,134 +222,154 @@ const BuySellForm = ({ advert, handleClose, handleConfirm, setErrorMessage, setI
+                                                <Localize i18n_default_text="Buyer's instructions" />
+                                            )}
+                                        </Text>
+                                        {description.split('\n').map((text, idx) => (
+                                            <Text key={idx} as='p' color='general' line_height='m' size='xs'>
+                                                {text || '-'}
+                                            </Text>
+                                        ))}
+                                    </div>
                                 </div>
-                            </div>
-                            <div className='buy-sell__popup-field-wrapper'>
-                                <Field name='amount'>
-                                    {({ field }) => (
-                                        <Input
-                                            {...field}
-                                            data-lpignore='true'
-                                            type='number'
-                                            error={errors.amount}
-                                            label={localize('Amount')}
-                                            hint={
-                                                <Localize
-                                                    i18n_default_text='Limits: {{min}}–{{max}} {{currency}}'
-                                                    values={{
-                                                        min: min_order_amount_limit_display,
-                                                        max: max_order_amount_limit_display,
-                                                        currency: account_currency,
-                                                    }}
-                                                />
-                                            }
-                                            className='buy-sell__popup-field'
-                                            trailing_icon={
-                                                <span className='buy-sell__popup-field--trailing'>
-                                                    {account_currency}
-                                                </span>
-                                            }
-                                            onChange={event => {
-                                                if (event.target.value === '') {
-                                                    setFieldValue('amount', '');
-                                                    setReceiveAmount(0);
-                                                    return;
+                                <div
+                                    className={classNames('buy-sell__modal-field-wrapper', {
+                                        'buy-sell__modal-field-wrapper--no-flex': isMobile(),
```

Comment:
Why not use a mixin?

Reply:
Not all mobile `buy-sell__modal-field-wrapper`s should be `no-flex`.